src/use_case_signup/SignupInputBoundary.java

```diff
@@ -0,0 +1,10 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
+
+package use_case_signup;
+
+public interface SignupInputBoundary {
+    void execute(SignupInputData var1);
```

Comment:
For readability purposes, could you change this to a more informative name, like `signupInputData`.